docs/docs/recipes.md

```diff
@@ -853,6 +853,138 @@ export const pageQuery = graphql`
 
+### Sourcing from WordPress
+
+#### Prerequisites
+
+- An existing [Gatsby site](/docs/quick-start/) with a `gatsby-config.js` and `gatsby-node.js` file
+
+#### Directions
+
+1. Install the `gatsby-source-wordpress` plugin by running the following command:
+
+```shell
+npm install gatsby-source-wordpress --save
+```
+
+2. Configure the plugin by modifying the `gatsby-config.js` file such that it includes the following:
+
+```JS:title=gatsby-config.js
+module.exports = {
+  ...
+  plugins: [
+    {
+      resolve: `gatsby-source-wordpress`,
+      options: {
+        // baseUrl will need to be updated with your wordpress source
+        baseUrl: `wpexample.com`,
+        protocol: `https`,
+        // is it hosted on wordpress.com, or self-hosted?
+        hostingWPCOM: false,
+        // does your site use the Advanced Custom Fields Plugin?
+        useACF: false
+      }
+    },
+  ]
+}
+```
+
+> > **Note:** Refer to the [`gatsby-source-wordpress` plugin docs](/packages/gatsby-source-wordpress/?=wordpre#how-to-use) to know more about configuring your plugins.
+
+3. Create a template component such as `src/templates/post.js` with the following code in it:
+
+```JS:title=post.js
+import React, { Component } from "react"
+import { graphql } from "gatsby"
+import PropTypes from "prop-types"
+
+class Post extends Component {
+  render() {
+    const post = this.props.data.wordpressPost
+
+    return (
+      <>
+        <h1>{post.title}</h1>
+        <div>{post.content}</div>
+      </>
+    )
+  }
+}
+
+Post.propTypes = {
+  data: PropTypes.object.isRequired,
+  edges: PropTypes.array,
+}
+
+export default Post
+
+export const pageQuery = graphql`
+  query($id: String!) {
+    wordpressPost(id: { eq: $id }) {
+      title
+      content
+    site {
+      siteMetadata {
+        title
+        subtitle
+      }
+    }
+  }
+`
+```
+
+4. Create dynamic pages for your Wordpress posts by pasting the following sample code in `gatsby-node.js`:
+
+```JS:title=gatsby-node.js
+const path = require(`path`)
+const slash = require(`slash`)
+
+exports.createPages = async ({ graphql, actions }) => {
+  const { createPage } = actions
+
+  // query content for WordPress posts
+  const result = await graphql(`
+    query {
+      allWordpressPost {
+        edges {
+          node {
+            id
+            slug
+          }
+        }
+      }
+    }
+  `)
+
+  const postTemplate = path.resolve(`./src/templates/post.js`)
+  result.data.allWordpressPost.edges.forEach(edge => {
+    createPage({
+      // `path` will be the url for the page
+      path: edge.node.slug,
+      // specify the component template of your choice
+      component: slash(postTemplate),
+      // In the ^template's GraphQL query, 'id' will be available
+      // as a GraphQL variable to query for this posts's data.
+      context: {
+        id: edge.node.id,
+      },
+    })
+  })
+}
+```
+
+5. Run `gatsby-develop` to see the newly generated pages and navigate through them. 
```

Comment:
That would be clearer, I agree! Although wouldn't it be better to keep it that way for consistency, since every step on the Recipes page that mentions running the server uses the same pattern? Let me know what you think.

Reply:
A) That's why I suggest to change all other commands on this page also. 

B) Or we postpone this change and do an "change it all on receipts" PR just for this change later.

Reply:
You are right. (B) would be the better way. Every PR should change only one thing. 

So go on with current style and ignore my comment

Reply:
Changing all the command patterns on the page would be a helpful addition that would lend more clarity to the viewers. However, we can open another PR just for that purpose as you suggested. Since this PR only addresses a specific issue, resolving the former through another PR would be easier to track and review. 
Thanks for pointing it out! 😄

Reply:
Open a nee issue for this: #1726

Reply:
Agreed that we should keep bigger decisions on formatting apart from this PR, which is focused on a WordPress recipe. @muescha I think that's the wrong link...?

Reply:
#17626